src/sdl2_compat.c

```diff
@@ -2559,37 +2559,11 @@ GestureAddTouch(const SDL_TouchID touchId)
     return &GestureTouches[GestureNumTouches++];
 }
 
-static int
-GestureDelTouch(const SDL_TouchID touchId)
-{
-    int i;
-
-    for (i = 0; i < GestureNumTouches; i++) {
-        if (GestureTouches[i].touchId == touchId) {
-            break;
-        }
-    }
-
-    if (i == GestureNumTouches) {
-        /* not found */
-        return -1;
-    }
-
-    SDL3_free(GestureTouches[i].dollarTemplate);
-    SDL3_zero(GestureTouches[i]);
-
-    GestureNumTouches--;
-    if (i != GestureNumTouches) {
-        SDL3_copyp(&GestureTouches[i], &GestureTouches[GestureNumTouches]);
-    }
-    return 0;
-}
-
 static GestureTouch *
 GestureGetTouch(const SDL_TouchID touchId)
 {
-    int i;
-    for (i = 0; i < GestureNumTouches; i++) {
+    int i = 0;
+    for (; i < GestureNumTouches; i++) {
```

Comment:
Why would you make this change? It's typical for loop variables to be initialized as part of the loop construct.

Reply:
I usually do it that way if it's done only once in a procedure like this one. Can revert that part if you want.

Reply:
Yes please.

Reply:
This looks good to me, once you've reverted this change.

Reply:
Done